scripting/setup/Installing-the-Windows-PowerShell-SDK.md

```diff
@@ -11,265 +11,216 @@ ms.assetid:  c3636b45-61aa-4720-85f0-58312c4fc8f9
-                </definition>
-                <definedTerm>AccessDBProviderSample05</definedTerm>
-                <definition>
-                  <para>Shows how to overwrite container methods to support calls to the Move-Item and Join-Path cmdlets. These methods should be implemented when the user needs to move items within a container and if the data store contains nested containers. The provider class in this sample derives from the <codeEntityReference>T:System.Management.Automation.Provider.NavigationCmdletProvider</codeEntityReference> class.</para>
-                </definition>
-                <definedTerm>AccessDBProviderSample06</definedTerm>
-                <definition>
-                  <para>Shows how to overwrite content methods to support calls to the Clear-Content, Get-Content, and Set-Content cmdlets. These methods should be implemented when the user needs to manage the content of the items in the data store. The provider class in this sample derives from the <codeEntityReference>T:System.Management.Automation.Provider.NavigationCmdletProvider</codeEntityReference> class, and it implements the <codeEntityReference>T:System.Management.Automation.Provider.IContentCmdletProvider</codeEntityReference> interface.</para>
-                </definition>
-              </definitionTable>
-            </content>
-          </section>
-        </sections>
-      </section>
-    </sections>
-  </section>
-  <relatedTopics />
-</developerConceptualDocument>
 
+The following topic describes how to install the PowerShell SDK on different versions of Windows.
+
+## Installing Windows PowerShell 3.0 SDK for Windows 8 and Windows Server 2012
+
+Windows PowerShell 3.0 is automatically installed with Windows 8 and Windows Server 2012. In addition, you can download and install the reference assemblies for Windows PowerShell 3.0 as part of the Windows 8 SDK. These assemblies allow you to write cmdlets, providers, and host programs for Windows PowerShell 3.0. When you install the Windows SDK for Windows 8, the Windows PowerShell assemblies are automatically installed in the reference assembly folder, in \Program Files (x86)\Reference Assemblies\Microsoft\WindowsPowerShell\3.0. For more information, see the [Windows 8 SDK download site](http://msdn.microsoft.com/windows/hardware/hh852363.aspx). Windows PowerShell code samples are also available on the Development Center. For more information, see the Desktop code sample page on the [Dev center site](http://code.msdn.microsoft.com/windowsdesktop/). 
```

Comment:
Can you, please, break paragraphs to use one sentence per line?
It makes reading raw text easier, also greatly improves subsequent editing, because git operates on a line-level.